dask_planner/src/sql/logical/sort.rs

```diff
@@ -0,0 +1,78 @@
+use crate::expression::PyExpr;
+
+use datafusion_expr::logical_plan::Sort;
+pub use datafusion_expr::{logical_plan::LogicalPlan, Expr};
+
+use crate::sql::exceptions::py_type_err;
+use pyo3::prelude::*;
+
+#[pyclass(name = "Sort", module = "dask_planner", subclass)]
+#[derive(Clone)]
+pub struct PySort {
+    sort: Sort,
+}
+
+impl PySort {
+    /// Returns if a sort expressions denotes an ascending sort
+    fn is_ascending(&self, expr: Expr) -> bool {
+        match expr {
+            Expr::Sort {
+                expr: _,
+                asc,
+                nulls_first: _,
+            } => asc,
```

Comment:
I was trying to figure out a way to use some of the utilities in expressions.rs instead of explicitly matching here but iiic we need to create a `PyExpr`, get the expr type and then match to this if the result is a `Sort` expr type which may not be preferred.